luminoso_api/json_stream.py

```diff
@@ -254,6 +260,13 @@ def _read_csv(reader, header, encode_fn):
         yield row_dict
 
 
+def convert_date(date_string, date_format):
+    if date_format != 'epoch':
+        return datetime.datetime.strptime(date_string, date_format).timestamp()
```

Comment:
If the date is in the wrong format--it's missing the time for a format that expects the time, or it has a time for a format that doesn't, or, god forbid, it has an extra space in it--then this will raise an exception and the entire json_stream will stop.  Would we rather wrap this in a "try: ...; except ValueError: return", so that an unparseable date is just skipped?